providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/grpc/EventStreamObserver.java

```diff
@@ -20,25 +20,25 @@
 @Slf4j
 @SuppressFBWarnings(justification = "cache needs to be read and write by multiple objects")
 class EventStreamObserver implements StreamObserver<EventStreamResponse> {
-    private final BiConsumer<Boolean, List<String>> stateConsumer;
+    private final BiConsumer<Boolean, List<String>> onConnectionEvent;
     private final Object sync;
     private final Cache cache;
 
-    private static final String CONFIGURATION_CHANGE = "configuration_change";
-    private static final String PROVIDER_READY = "provider_ready";
+    public static final String CONFIGURATION_CHANGE = "configuration_change";
```

Comment:
[nit] maybe we should put those into an own class/enum

Reply:
Done: https://github.com/open-feature/java-sdk-contrib/pull/967/commits/e61a4628aa175276490dbcabf196023f034a303c